gdx-freetype-jtransc/src/com/badlogic/gdx/graphics/g2d/freetype/FreeType.java

```diff
@@ -0,0 +1,710 @@
+
+		public Bitmap getBitmap() {
+			if (bitmap == null) {
+				bitmap = slot.getBitmap();
+			}
+			return bitmap;
+		}
+
+		public int getLeft() {
+			return toInt(slot.getMetrics().getHoriBearingX());
+		}
+
+		public int getTop() {
+			return toInt(slot.getMetrics().getHoriBearingY());
+		}
+	}
+
+	@HaxeImports(
+		"import lime.graphics.Image;"
+	)
+	@HaxeAddMembers(
+		"var image: Image;"
+	)
+	public static class Bitmap {
+		private static Face face;
+
+		Bitmap(Face face) {
+			this.face = face;
+			init(face);
+		}
+
+		@HaxeMethodBody(
+			"image = p0.font.renderGlyph(p0.glyph, p0.pixelHeight);"
+		)
+		private void init(Face face) {
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? image.height : 1;"
+		)
+		public int getRows() {
+			throw new RuntimeException("FreeType.Bitmap.getRows(): Not implemented.");
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? image.width : 10;"
+		)
+		public int getWidth() {
+			throw new RuntimeException("FreeType.Bitmap.getWidth(): Not implemented.");
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? (image.buffer.width * image.buffer.bitsPerPixel) : 10;"
+		)
+		public int getPitch() {
+			throw new RuntimeException("FreeType.Bitmap.getPitch(): Not implemented.");
+		}
+
+		public ByteBuffer getBuffer() {
+			return ByteBuffer.wrap(getData());
+		}
+
+		@HaxeMethodBody(
+			"if(image == null) " +
+				" 	return new JA_B(10);" +
+				"var bytes = image.data.toBytes();" +
+				"return new JA_B(bytes.length, bytes);"
+		)
+		private byte[] getData() {
+			return new byte[0];
+		}
+
+		public Pixmap getPixmap(Format format, Color color, float gamma) {
+			Pixmap pixmap = new Pixmap(getWidth(), getRows(), Format.RGBA8888);
+			ByteBuffer buffer = pixmap.getPixels();
+			byte[] src = getData();
+			byte[] dest = new byte[src.length * 4];
+
+			byte r = (byte) (color.r * 0xFF);
+			byte g = (byte) (color.g * 0xFF);
+			byte b = (byte) (color.b * 0xFF);
+			int a = ((int) (color.a * 0xFF)) & 0xFF;
+
+			for (int i = 0; i < dest.length; ) {
+				// Use the specified color for RGB, blend the FreeType bitmap with alpha.
+				float alpha = (src[i >> 2] & 0xff) / 255f;
+				alpha = (float) Math.pow(alpha, gamma); // Inverse gamma.
+				dest[i++] = r;
+				dest[i++] = g;
+				dest[i++] = b;
+				dest[i++] = (byte) (a * alpha);
+			}
+
+			BufferUtils.copy(dest, 0, buffer, dest.length);
+			return pixmap;
+		}
+
+		@HaxeMethodBody(
+			"return 0;"
+		)
+		public int getNumGray() {
+			throw new RuntimeException("FreeType.Bitmap.getNumGray(): Not implemented.");
+		}
+
+		@HaxeMethodBody(
+			"return 0;"
+		)
+		public int getPixelMode() {
+			throw new RuntimeException("FreeType.Bitmap.getPixelMode(): Not implemented.");
+		}
+	}
+
+	@HaxeImports(
+		"import lime.text.GlyphMetrics;"
+	)
+	@HaxeAddMembers(
+		"public var metrics: GlyphMetrics;"
+	)
+	public static class GlyphMetrics {
+		private Face face;
+
+		GlyphMetrics(Face face) {
+			this.face = face;
+			init(face);
+		}
+
+		@HaxeMethodBody(
```

Comment:
I will work on later versions of jtransc so you can directly call haxe classes. Maybe even creating dummy classes directly.